napari/utils/patcher.py

```diff
@@ -0,0 +1,212 @@
+"""Patch functions and methods. Our perfmon system using this to patch in
+perf_timers, but this can be used for any type of patching.
+
+See patch_callables() below as the main entrypoint.
+"""
```

Comment:
It was more as functions at first but it grew in this direction. I didn't think of using `sys.modules` that would simplify things.  And then I didn't realize `module.__name__` is the "path" in my terms, it's `napari.utils.perf` or whatever.

Any class can be mechanically turned into a module where the attributes become module globals and the methods become functions. I don't think that's automatically a good idea just because it's possible. But using `sys.modules` and `module` here does make sense. And if there just was no point at all to some of the classes it might make sense to get rid of them.

Reply:
New version is all functions. Using sys.modules and module.__name__ and then unwinding some things, no more needs for the classes.